contents/teams/surveys/objectives.mdx

```diff
@@ -1,61 +1,61 @@
+# 📣 Goal 2: Marketing
 
-# 🎨 Goal 3: Survey customization
+> Why: Increasing awareness and adoption of Surveys through focused outreach.
 
-A lot of people use the API to tweak their surveys. Let’s bring those use cases into the product.
+> Who: <TeamMember name="Manoel Aranda Neto" photo /> and the [Brand & Vibes Team](/teams/brand-vibes).
+
+
+We're launching two major updates this quarter, new pricing and deeper AI integration. Our core focus is to drive usage, validate product–market fit, and grow our customer base and revenue. However, many users still aren’t aware that surveys are part of our offering. That’s why we’re doubling down on marketing, to reach existing users who haven’t tried Surveys yet, and to attract entirely new customers who could benefit from them.
+
+# 🛠️ Goal 3: Customization
+
+> Why: Expanding the flexibility of Surveys to meet diverse user needs.
+
+> Who: <TeamMember name="Lucas Faria" photo />
+
+
+Surveys already offer a high degree of customization, but our customers continue to ask for even more flexibility. Last quarter, we introduced several enhancements in this area, and we’re now taking it further by addressing the remaining gaps. This update focuses on expanding customization options to better meet the diverse needs of our users and give them even more control over how surveys look and behave. With these improvements, users will have more ways to tailor the survey experience to fit their brand, product flow, and audience.
 
 - [Better wait period controls](https://github.com/PostHog/posthog/issues/30379)
 - Alignment customization [here](https://github.com/PostHog/posthog/issues/28479) and [here](https://github.com/PostHog/posthog/issues/28477)
 - [Inline surveys](https://github.com/PostHog/posthog/issues/30097)
 - [Add icons](https://github.com/PostHog/posthog/issues/28478)
 - [Templated strings](https://github.com/PostHog/posthog/issues/26620)
-- [Single-response submit](https://github.com/PostHog/posthog/issues/30196)
+- ...
+- This is not an exhaustive list
+
+# 🧩 Goal 4: Deeper integration with Surveys and AI
+
+> Why: Making survey creation smarter and more contextual with AI.
+
+> Who: <TeamMember name="Lucas Faria" photo />, <TeamMember name="Manoel Aranda Neto" photo /> and <TeamMember name="Ioannis Josephides" photo />
 
-# 🧩 Goal 4: Templates, Templates, Templates
 
-Right now, duplicating and customizing surveys across projects/environments isn’t great. Let’s fix that.
+We're making survey creation more intelligent and contextual. Building on last quarter’s AI-assisted survey creation, we’re going further by deeply integrating AI across the survey workflow. For example, when creating a new experiment, you'll be able to instantly generate a relevant survey with AI, recommending the most appropriate questions based on context from the experiment. The goal is to make survey creation smarter, faster, and more tailored to what you’re working on, right when you need it.
```

Comment:
I feel like we should be specific about what we are going to build here. You gave the experiments example - will that be the one? Or will it be something else? This just doesn't feel concrete enough, and non-concrete goals can tend to get ignored.